FrostingContextExtensions.cs

```diff
@@ -0,0 +1,27 @@
+namespace BuildScripts;
+
+public static class FrostingContextExtensions
+{
+    private static readonly ProcessSettings _processSettings;
+
+    static FrostingContextExtensions() => _processSettings = new ProcessSettings();

```

Comment:
Just do:
```cs
private static readonly ProcessSettings _processSettings = new();
```

Reply:
Done in [99399d0](https://github.com/MonoGame/MonoGame.Tool.BuildScripts/pull/4/commits/99399d06404772193246100af83ad883db308eef)